src/main/java/com/sap/oss/phosphor/fosstars/data/github/CodeOfConductGuidelineInfo.java

```diff
@@ -153,9 +153,7 @@ protected ValueSet fetchValuesFor(GitHubProject project) throws IOException {
 
     for (String path : knownCodeofConductGuidelineFiles) {
       Optional<String> content = repository.readTextFrom(path);
-      if (content.isPresent()) {
-        return content;
-      }
+      return content;
```

Comment:
I just wanted to know your thoughts if there was a better way. By removing this if condition, it becomes even worse in a for loop.

Now for loop will run only once. Please revert it back.

Reply:
Reverted.